internal/validations/clusterlogforwarder/validate_clusterlogforwarderspec.go

```diff
@@ -258,6 +270,22 @@ func verifyGoogleCloudLogging(gcl *loggingv1.GoogleCloudLogging) bool {
 	return i == 1
 }
 
+func verifyAzureMonitorLog(name string, azml *loggingv1.AzureMonitor) (bool, status.Condition) {
```

Comment:
Let's start cleaning up our validations.  Please move to validations/clf/outputs

Reply:
done